src/figuresClasses.ts

```diff
@@ -1,19 +1,91 @@
-export interface Figure {
+type Colors = 'red' | 'green' | 'blue';
+
+enum Shapes {
```

Comment:
Don't name type in the plural. Type can be only one triangle or circle or rectangle, no several
```suggestion
enum Shape {
```

Reply:
Fix this in all places